docs/DeveloperGuide.adoc

```diff
@@ -296,6 +296,29 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 (For all use cases below, the *System* is `Cooking Papa` and the *Actor* is the `user`, unless specified otherwise)
 
+[source,shell]
```

Comment:
I think this causes syntax highlighting, might want to remove. If I'm not wrong, you can try replacing the "----" with "...."

Reply:
Ok noted. Changed accordingly. Thanks!

Reply:
can simply change to ``` too